icons/syncthing.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Syncthing</title><path d="M12 0A12 12 0 0 0 0 12a12 12 0 0 0 12 12 12 12 0 0 0 12-12A12 12 0 0 0 12 0m0 2.412c3.115 0 5.886 1.5 7.63 3.814a1.834 1.834 0 0 1 1.564 3.163 9.52 9.52 0 0 1-1.813 8.62c.128.28.189.595.162.921a1.85 1.85 0 0 1-2.003 1.677 1.86 1.86 0 0 1-.877-.322A9.5 9.5 0 0 1 12 21.505c-3.84 0-7.154-2.277-8.668-5.552a1.86 1.86 0 0 1-.879-.253c-.858-.51-1.145-1.635-.634-2.514.165-.276.392-.492.654-.642a10 10 0 0 1-.02-.585c0-5.265 4.282-9.547 9.547-9.547m0 1.227a8.31 8.31 0 0 0-8.312 8.683q.335.051.646.23c.323.2.565.484.714.806l6.98-.645c.11-.435.38-.828.79-1.08a1.82 1.82 0 0 1 1.916-.042l3.5-2.907a1.83 1.83 0 0 1 .323-1.844A8.3 8.3 0 0 0 12 3.639m7.027 5.981-3.499 2.906a1.83 1.83 0 0 1-.231 1.723l1.901 2.743q.32-.092.67-.065.357.038.66.192a8.28 8.28 0 0 0 1.792-5.16c0-.723-.094-1.423-.268-2.092a1.84 1.84 0 0 1-1.025-.247m-6.888 4.314-6.97.644a1.8 1.8 0 0 1-.657 1 8.3 8.3 0 0 0 7.488 4.7 8.3 8.3 0 0 0 3.936-.992 1.85 1.85 0 0 1 .258-1.592L14.29 14.95a1.84 1.84 0 0 1-2.151-1.016"/></svg>
```

Comment:
I'm seeing a single small differences:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/d4212735-3327-44b2-9d0b-f7da1caf9925)

Reply:
Thanks for pointing that out, somehow missed that. Could you take a look now? Not seeing any difference snow on my end.